invenio_oauthclient/contrib/orcid.py

```diff
@@ -56,11 +61,11 @@
            consumer_secret="<CLIENT SECRET>",
        )
 
-4. Now go to ``CFG_SITE_SECURE_URL/oauth/login/orcid/`` (e.g.
+4. Now go to ``CFG_SITE_URL/oauth/login/orcid/`` (e.g.
```

Comment:
please remove mentions of `CFG_SITE_*` variables and use just the example localhost.